src/Classes/LinkRenderers/NavLinkRenderer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace LaraZeus\Sky\Classes\LinkRenderers;
+
+use Illuminate\Database\Eloquent\Model;
+
+abstract class NavLinkRenderer
+{
+    public static string $renders;
```

Comment:
Each implementing class except the default should set this as it will be used to key the map of renderers. This value should match the `slug` provided by any custom item type added to the site for navigation.

Reply:
maybe `renders-key` if I got it right :)